src/main/resources/ui/repository/WorkflowRepository.js

```diff
@@ -12,6 +12,18 @@ let WorkflowRepository = new function () {
         );
```

Comment:
`this.getWorkflows` seems to be unused. Did you forget to remove it or did you leave it on purpose?